src/pages/Comment.tsx

```diff
@@ -0,0 +1,105 @@
+import {FC, useState} from "react";
+import {useSetAppTitle} from "../hooks";
+import {
+    Button,
+    FormikField,
+    Label,
+    Stack,
+    TextArea,
+    useDeskproAppClient,
+    useInitialisedDeskproAppClient
+} from "@deskpro/app-sdk";
+import {useStore} from "../context/StoreProvider/hooks";
+import {IntlProvider} from "react-intl";
+import { Formik } from "formik";
+import {addIssueComment, getIssueComments} from "../context/StoreProvider/api";
+
+interface CommentFormValues {
+    comments: string;
+}
+
+interface CommentProps {
+    issueKey: string;
+}
+
+export const Comment: FC<CommentProps> = ({ issueKey }: CommentProps) => {
+    const {client} = useDeskproAppClient();
+    const [, dispatch] = useStore();
+    const [isLoading, setIsLoading] = useState<boolean>(false);
+
+    useSetAppTitle("Add Comment");
+
+    useInitialisedDeskproAppClient((client) => {
+        client.deregisterElement("edit");
+        client.deregisterElement("addIssue");
+        client.deregisterElement("viewContextMenu");
+    }, [issueKey]);
+
+    const initialValues = {
+        comments: "",
+    } as CommentFormValues;
```

Comment:
```suggestion
    const initialValues: CommentFormValues = {
        comments: "",
    };
```

It definately a nit in this case, but `as CommentFormValues` is a form of assertion, its much safer to use `: CommentFormValues` instead.